File: 9. Palindrome Number/9.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        a = x
        if a < 0:
            return False
        if a == 0:
            return True
        lst = []
        while x > 0:
            lst.append(str(x % 10))
            x = x // 10
        b = int("".join(lst))

        if a == b:
            return True
        else:
            return False

File: 9. Palindrome Number/9.js
function isPalindrome(x) {
    let a = x;
    if (a < 0) {
        return false;
    }
    if (a === 0) {
        return true;
    }
    let lst = [];
    while (x > 0) {
        lst.push(String(x % 10));
        x = Math.floor(x / 10);
    }
    let b = parseInt(lst.join(''));
    
    return a === b;
}